### PR TITLE
Updating example in Readme for cases when sudo needs to be used

### PR DESCRIPTION
**Pull request by gtarnaras**

### CODE REVIEW

**Comment by asachs01** [comment]
Oof. This is a bit of a hot topic when it comes to Sensu. We absolutely need to note this on the remediation handler readme. There's been a larger conversation in the Sensu community around adding the `sensu` user to sudoers and what the potential security implications are for carte-blanche access to a system. That is to say, there's room for us to be prescriptive in our recommendations for sudo use (i.e., ensuring that if we have a note like this in a readme that it's linked to a recommended best practice that properly scopes the `sensu` user's access. 

There's also a distinction between Sensu having access to system commands, versus having access to executables that might require sudo (I'm drawing a blank on those commands atm). I'm not _quite_ sure of where everything's landed so far, but IIRC the lack of predictable asset names means that if you're operating from the principle of least privilege, you'd have to add the Sensu to sudoers with the path to the command, which includes the SHA512sum as part of the path, and that's kind of unwieldy. 

All that to say, I'm totally in favor of adding the note re: sudoers, but I'd like to get feedback from @cwjohnston & @calebhailey to see if there's more a note like this can do to ensure that folks don't open themselves up to a potential security snafu.

File: README.md [hunk 1]
```diff
@@ -175,6 +175,9 @@ spec:
   subscriptions: []
 ```
 
+**NOTE:** Please note that sensu user needs elevated permissions to run commands with sudo, which require root privileges.
```
Comment (cwjohnston) [inline README.md]:
```suggestion
**NOTE:** Many remediation commands require root privileges. You can provide the `sensu` user with elevated permissions using sudo. Configuring sudo outside the scope of this document. 
```

File: README.md [hunk 2]
```diff
@@ -175,6 +175,9 @@ spec:
   subscriptions: []
 ```
 
+**NOTE:** Please note that sensu user needs elevated permissions to run commands with sudo, which require root privileges. 
+As such, sensu user needs to be added to the sudoers file.
```
Comment (cwjohnston) [inline README.md]:
```suggestion
```

**Review by cwjohnston — CHANGES_REQUESTED** [review]

**Comment by calebhailey** [comment]
Closing in favor of this approach: https://github.com/calebhailey/sensu-remediation-handler/tree/rewrite#privilege-escalation 

Working on (finally!) getting this merged into this repo; see #18 which closes most of the open issues in this repository.

**Pull request closed without merging**